src/m365/aad/commands/user/user-set.spec.ts

```diff
@@ -291,4 +388,64 @@ describe(commands.USER_SET, () => {
     } as any);
     assert(loggerLogSpy.notCalled);
   });
+
+  it('creates Azure AD user and set its manager by id', async () => {
+    sinon.stub(request, 'patch').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${formatting.encodeQueryParameter(userPrincipalName)}`) {
+        return userResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    sinon.stub(request, 'put').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${userPrincipalName}/manager/$ref`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await assert.doesNotReject(command.action(logger, { options: { userPrincipalName: userPrincipalName, displayName: displayName, managerUserId: managerUserId } }));
+  });
+
+  it('creates Azure AD user and set its manager by user principal name', async () => {
+    sinon.stub(request, 'patch').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${formatting.encodeQueryParameter(userPrincipalName)}`) {
+        return userResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    sinon.stub(request, 'put').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${userPrincipalName}/manager/$ref`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await assert.doesNotReject(command.action(logger, { options: { userPrincipalName: userPrincipalName, displayName: displayName, managerUserName: managerUserName } }));
```

Comment:
`assert.doesNotReject` is not really needed here because when an error is thrown, the test promise will fail.